updating-bazel.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: contribute
+title: Updating Bazel
+---
+
+# Updating Bazel
+
+Bazel defined a [backwards compatibility policy](https://docs.bazel.build/versions/master/backward-compatibility.html)
+(see [https://www.bazel.build/breaking-changes-guide.html](https://www.bazel.build/breaking-changes-guide.html) for how
+to roll out incompatible changes if you are the author of one).
```

Comment:
Don't show the url. You can put the link on "how to roll out incompatible changes" instead.

Reply:
will fix.

Reply:
Addressed in https://github.com/bazelbuild/bazel-website/pull/197